src/extensions/environments/environment.ts

```diff
@@ -1,5 +1,10 @@
+import { Tester } from './../tester';
+import { Compiler } from '../compile';
```

Comment:
@davidfirst This make a cyclic deps between evns and compiler/tester
we should extract the interfaces to the general types I think (or use import type at least once we support it)

Reply:
@GiladShoham , yeah, I was thinking about it. But it's only for the type. Once we migrate "prettier" to v2, we can update Babel to the latest version that supports `import type`. Then, you won't see any circular.